chef_master/source/config_rb_server_optional_settings.rst

```diff
@@ -809,6 +815,9 @@ This configuration file has the following settings for ``oc-id``:
 ``oc_id['port']``
    The port on which the service is to listen. Default value: ``9090``.
 
+``oc_id['sql_connection_user']``
```

Comment:
A nil value means the system uses the string 'oc_id', right? So if they were using azure they'd need to fully define this.

Reply:
Reworded this.  Let me know what you think.